src/pages/Gallery/components/CreateGallery.tsx

```diff
@@ -0,0 +1,103 @@
+import { Box, Button, MenuItem } from '@mui/material';
+import { makeStyles } from 'makeStyles';
+import { useState } from 'react';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import { Gallery, GalleryRequired } from 'types';
+
+import { useEvents } from 'hooks/Event';
+import { useCreateAlbum } from 'hooks/Gallery';
+import { useSnackbar } from 'hooks/Snackbar';
+
+import Select from 'components/inputs/Select';
+import SubmitButton from 'components/inputs/SubmitButton';
+import TextField from 'components/inputs/TextField';
+import { ImageUpload } from 'components/inputs/Upload';
+import { BannerButton } from 'components/layout/Banner';
+import Dialog from 'components/layout/Dialog';
+
+const useStyles = makeStyles()((theme) => ({
+  margin: {
+    margin: theme.spacing(2, 0, 1),
+    borderRadius: theme.shape.borderRadius,
+    overflow: 'hidden',
+  },
+}));
+type FormValues = Omit<Gallery, 'slug'>;
+
+const CreateAlbum = () => {
+  const [acceptedFileTypesOpen, setAcceptedFileTypesOpen] = useState<boolean>(false);
+  const { handleSubmit, register, watch, formState, setValue, control } = useForm<FormValues>();
+  const acceptedFileTypes = ['jpg', 'png'];
+  // const events = useMemo(() => (data ? data.pages.map((page) => page.results).flat() : []), [data]);
+  const createAlbum = useCreateAlbum();
+  const { classes } = useStyles();
+  const showSnackbar = useSnackbar();
+
+  const submit: SubmitHandler<FormValues> = async (data) => {
+    const gallery = {
+      ...data,
+      slug: '_',
+      title: data.title,
+      description: data.description,
+      image: data.image,
+      image_alt: data.image_alt,
+    } as GalleryRequired;
+    await createAlbum.mutate(gallery, {
+      onSuccess: () => {
+        showSnackbar('Albumet ble lagt til', 'success');
+      },
```

Comment:
Dialogen for opprettelse av album bør lukkes ved opprettelse, eller man bør redirigeres til det nyopprettede albumet
```suggestion
      onSuccess: (data) => {
        showSnackbar('Albumet ble lagt til', 'success');
        closeDialog();
        // eller:
        navigate(`${URLS.gallery}${data.id}/`)
      },
```